scraps/release.sh

```diff
@@ -50,7 +50,7 @@ function updateWithRelease {
 
   git checkout -b "version-$versionNext"
   python version.py "$versionNext"
-  git add bower.json package.json src/chrome/app/manifest.json src/chrome/extension/manifest.json src/firefox/package.json
+  git add bower.json package.json src/chrome/app/manifest.json src/chrome/extension/manifest.json src/chrome/app/managed_policy_schema.json src/firefox/package.json
```

Comment:
This and the other scrap change were just added by finding wherever manifest.json was referenced within potential build processes since I don't know that code base as well. I'm removing this though since it doesn't seem to be necessary, per your comment.